index.html

```diff
@@ -4,12 +4,12 @@
 <head>
     <!-- ========== Meta Tags ========== -->
     <meta charset="UTF-8">
-    <meta name="description" content="Evento -Event Html Template">
-    <meta name="keywords" content="Evento , Event , Html, Template">
+    <meta name="description" content="Thailand .NET Conf 2018">
+    <meta name="keywords" content="dotnet, .NET, conference, Thailand">
     <meta name="author" content="ColorLib">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <!-- ========== Title ========== -->
-    <title> dotnet conf - Thailand</title>
+    <title>dotnet conf - Thailand</title>
```

Comment:
Change `dotnet conf` to `.NET Conf` for consistency (and that is also how MS writes it)